app/gui/src/dashboard/components/dashboard/column/LabelsColumn.tsx

```diff
@@ -42,6 +50,24 @@ export default function LabelsColumn(props: column.AssetColumnProps) {
     category.type !== 'trash' &&
     (self?.permission === permissions.PermissionAction.own ||
       self?.permission === permissions.PermissionAction.admin)
+  const temporarilyAddedLabels = useStore(driveStore, ({ labelsDragPayload, dragTargetAssetId }) =>
```

Comment:
The add labels modal is broken - the submit is always disabled. Also I'm not a fan of having optimistic state, can we just add a loader instead?

Reply:
this is not optimistic state, this is drag preview (i think it's useful for when labels are about to be removed - although if you think shift+drag for that is more intuitive then that can definitely be changed (and would greatly simplify the code for handling drag and drop))